sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/policy/SasTokenCredentialPolicy.java

```diff
@@ -1,49 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.storage.common.implementation.policy;
-
-import com.azure.core.http.HttpPipelineCallContext;
-import com.azure.core.http.HttpPipelineNextPolicy;
-import com.azure.core.http.HttpResponse;
-import com.azure.core.http.policy.HttpPipelinePolicy;
-import com.azure.core.util.CoreUtils;
-import com.azure.core.util.logging.ClientLogger;
-import com.azure.storage.common.implementation.credentials.SasTokenCredential;
-import reactor.core.publisher.Mono;
-
-import java.net.MalformedURLException;
-import java.net.URL;
-
-/**
- * Policy that adds the SAS token to the request URL's query.
- */
-public final class SasTokenCredentialPolicy implements HttpPipelinePolicy {
```

Comment:
@alzimmermsft should we preserve this in case someone updates "common" version without bumping "blobs" ?

FYI @rickle-msft

Reply:
Yeah, this is worth keeping around for a while as it is hidden internally but removal could result in a functional break in the case you mentioned.